src/Phinx/Util/Util.php

```diff
@@ -221,4 +221,27 @@ public static function glob($path)
     {
         return glob($path, defined('GLOB_BRACE') ? GLOB_BRACE : 0);
     }
+
+    /**
+     * Takes the path to a php file and attempts to include it if readable
+     */
+    public static function loadPhpFile($filename)
+    {
+        $filePath = realpath($filename);
+
+        /**
+         * I lifed this from phpunits FileLoader class
+         *
+         * @see https://github.com/sebastianbergmann/phpunit/pull/2751
+         */
+        $isReadable = @\fopen($filePath, 'r') !== false;
```

Comment:
This is done intentionally see comment above this code block. I took this from how phpunit's [Filelocator class](https://github.com/sebastianbergmann/phpunit/blob/master/src/Util/FileLoader.php#L37) does it.

It was changed from `is_readable` to this because of https://github.com/sebastianbergmann/phpunit/pull/2751